ee/clickhouse/models/entity.py

```diff
@@ -0,0 +1,27 @@
+from typing import Any, Dict, Tuple
+
+from ee.clickhouse.models.action import format_action_filter
+from ee.clickhouse.models.property import parse_prop_clauses
+from posthog.constants import TREND_FILTER_TYPE_ACTIONS
+from posthog.models.entity import Entity
+
+
+def get_entity_filtering_params(
+    entity: Entity, team_id: int, table_name: str = "", *, with_prop_filters: bool = False
+) -> Tuple[Dict, Dict]:
+    params: Dict[str, Any]
+    content_sql_params: Dict[str, str]
+    if entity.type == TREND_FILTER_TYPE_ACTIONS:
+        action = entity.get_action()
+        action_query, params = format_action_filter(action, table_name=table_name)
+        content_sql_params = {"entity_query": "AND {action_query}".format(action_query=action_query)}
```

Comment:
I think you can have extra property filters on actions too that should be handled right?